docs/ensuring-repro/docker/updating-images.md

```diff
@@ -0,0 +1,21 @@
+# Updating Docker images
+
+Over the course of module development, new dependencies may get added or removed, thereby changing the [module's software environment](../managing-software/index.md).
+As modules progress to maturity, there are important considerations to bear in mind when updating the module's software environment.
+
+In particular, over the course of module development, the Data Lab will activate two [GitHub Action (GHA) workflows](../workflows/index.md) to ensure module reproducibility:
+
+- A GHA workflow to [build and push the module's Docker image](#STUB_LINK../workflows/docker-build-gha.md) to the OpenScPCA Docker Registry
+    - This GHA automatically runs when pull requests (PRs) that modify a module's environment are merged into `main`
+- A GHA workflow to [run the module using test data](../workflows/run-module-gha.md)
+    - This GHA automatically runs on PRs that modify any module code
+    - Once the module's Docker image has been pushed to the OpenScPCA Docker Registry, this GHA will pull that Docker image to build a runtime environment.
+    Therefore, for this GHA to succeed, it must use the most up-to-date Docker image.
```

Comment:
I feel like there should be a section on_how_ Docker images are updated and used which would include this information and then the second section should be how to actually update it. So I would break this out and talk about this in it's own section.